packages/Molecules/Pagination/index.vue

```diff
@@ -1,9 +1,17 @@
 <template>
-  <div class="rpl-pagination">
+  <nav class="rpl-pagination" role="navigation" :aria-label="label || 'search'">
     <ol class="rpl-pagination__list">
-      <li v-for="n in visibleStepRange" :key="n" class="rpl-pagination__list-item">
-        <button v-if="n !== currentStep" @click="gotoStep(n)" class="rpl-pagination__step"><span>Go to step </span>{{ n }}</button>
-        <span v-else class="rpl-pagination__step-current">{{ n }}</span>
+      <li v-for="n in visibleStepRange" :aria-current="n === currentStep ? 'page' : false" :aria-disabled="n === currentStep ? 'true' : false" :key="n" class="rpl-pagination__list-item">
+        <button
+          @click="gotoStep(n)"
+          class="rpl-pagination__step"
+          :class="{'rpl-pagination__step-current' : n === currentStep}"
+          :disabled="n === currentStep"
+          >
+          <span class="rpl-pagination__step-label">{{n !== currentStep ? 'Go to step' : 'current page'}}</span>
```

Comment:
I think the AC requested 'Go to step' be read as 'Go to page'.

Reply:
Added this as a prop because you might want to use pagination for items or cards instead of pages.